client/components/button/index.jsx

```diff
@@ -14,7 +14,9 @@ export default class Button extends PureComponent {
 		scary: PropTypes.bool,
 		type: PropTypes.string,
 		href: PropTypes.string,
-		borderless: PropTypes.bool
+		borderless: PropTypes.bool,
+		target: PropTypes.string,
+		rel: PropTypes.string
```

Comment:
Might be argued that these be included in the `else` block in `render` to push into `omitProps` for the button case, since they're not valid attributes for a button. This assumes someone renders as a button with `rel` and `target` but not `href`, which is unlikely, be "possible".